journalism_workflow/static/journalism_workflow/photographer/partials/photographer.html

```diff
@@ -0,0 +1,80 @@
+<section class="section-panel">
+  <div class="container-fluid">
+    <div class="row section-header">
+      <div class="col-lg-12 col-md-12 col-sm-12">
+        <h3>
+	  Take and upload photos for the article
+        </h3>
+      </div>
+    </div>
+
+    <!-- Titles for the two columns -->
+    <div class="row padded">
+      <div class="col-lg4 col-md-4 col-sm-4">
+	<h3>Editor's Story Plan</h3>
+      </div>
+
+      <div class="col-lg-8 col-md-8 col-sm-8">
+	<h3>Photographer Instructions</h3>
+      </div>
+
+    </div>
+
+    <div class="row padded">
+
+      <!-- Form with Editor's story plan -->
+      <div class="col-lg-4 col-md-4 col-sm-4">
+	<form class="form-horizontal text-right">
+
+	  <!-- Who was involved in the story -->
+	  <div class="form-group">
+	    <label for="whoInput" class="col-sm-2 col-md-2 col-lg-2 control-label text-right">Who</label>
+	    <div class="col-sm-10 col-md-10 col-lg-10">
+	      <input type="text" class="form-control" id="whoInput" value="{{who}}" disabled/>
```

Comment:
we do that.  then we'd do `value="{{vm.who}}"`  see https://github.com/unlimitedlabs/orchestra/blob/master/orchestra/static/orchestra/dashboard/partials/dashboard.html as an example.